src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +35,16 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    public static void convertToLowerCase(Collection<String> strings)//This can use for any other Collection data type: eg.Set
+    {
+        String[] stringsArray = strings.toArray(new String[0]);
+        for (int i=0; i<stringsArray.length; ++i)
```

Comment:
Coding standard requires space around operators, i.e. `for (i = 0; i < 10; i++) {` ← Good `for(i=0;i<10;i++){` ← Bad